selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/MacrosCommandsEditorTest.java

```diff
@@ -101,7 +101,6 @@ public void checkCommandMacrosIntoCommandLine() {
     commandsEditor.selectMacroCommand("${current.class.fqn}");
     commandsEditor.typeTextIntoEditor(Keys.ARROW_DOWN.toString());
     commandsEditor.typeTextIntoEditor(Keys.SPACE.toString());
-    commandsEditor.waitMacroCommandIsSelected("${current.project.path}");
```

Comment:
Why did you decide to remove this verification?

Reply:
It hard to check a selection now